_lib/wordpress_contact_processor.py

```diff
@@ -57,6 +57,10 @@ def process_contact(contact):
     convert_custom_field(contact, 'state')
     convert_custom_field(contact, 'zip_code')
     convert_custom_field(contact, 'addr_desc')
+    convert_custom_field(contact, 'email_0_addr', index=0, new_attribute='email_addr')
```

Comment:
I don't think we need to use the index or new_attribute arguments for lines 60-63.  From what I'm seeing in the [current output](http://www.consumerfinance.gov/api/get_posts/?post_type=contact&count=50), the only field that contains a list with more than one item is web_0.  Also, I don't see any instance of something_2_something, it looks like 0 and 1 only.

I think we can remove lines 45, 46, 47, 49, 50.  Then remove index/new_attribute arguments from all besides web_0.  Line 61 would also have to be changed to `convert_custom_field(contact, 'email_0_desc')`.